src/Humanizer.Tests.Shared/Localisation/fr/TimeSpanHumanizeTests.cs

```diff
@@ -155,6 +155,30 @@ public void MillisecondsToWords(int ms, string expected)
             Assert.Equal(expected, actual);
         }
 
+        [Theory]
+        [InlineData(4, false, "4 jours")]
+        [InlineData(23, false, "3 semaines")]
+        [InlineData(64, false, "2 mois")]
+        [InlineData(367, true, "un an")]
+        [InlineData(750, true, "deux ans")]
+        public void Age(int days, bool toWords, string expected)
+        {
+            var actual = TimeSpan.FromDays(days).ToAge(toWords: toWords);
```

Comment:
"12 mois / 24 mois", pour l'âge on compte en mois le plus souvent au début.


For babies less than 2 years old, we count in months instead of year in France.

Reply:
I think this is fine for start. Can be done as an improvement later.